stake-pool/cli/src/main.rs

```diff
@@ -1061,31 +1063,93 @@ struct WithdrawAccount {
     pool_amount: u64,
 }
 
+fn sorted_accounts<F>(
+    validator_list: &ValidatorList,
+    stake_pool: &StakePool,
+    get_account: F,
```

Comment:
nit: can we call this `get_pubkey`? `get_account` makes it sound like it's actually fetching an account